db/schema.rb

```diff
@@ -24,6 +24,22 @@
     t.integer "user_id"
   end
 
+  create_table "roles", force: :cascade do |t|
+    t.string "name"
+    t.string "slug"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "user_roles", force: :cascade do |t|
```

Comment:
Roll back this migration and push. I don’t want to see this in this PR

Reply:
ok